### PR TITLE
Ballet: version 1.100 added

### PR DESCRIPTION
**Pull request by RosaWagner**
Taken as upstream repo <https://github.com/Omnibus-Type/Ballet> at commit <https://github.com/Omnibus-Type/Ballet/commit/bd239d606894c2139ad997b60d2141bee7856817>.

The fontbakery Error probably comes from the fact that the font only have a optical size axis.

### CODE REVIEW

**Comment by m4rc1e** [comment]
Thanks!

In the future, we need to use the `gftools packager`.  Using it means we won't ever have to copy a file manually again. It's similar to symlinks.

**Comment by RosaWagner** [comment]
## Fontbakery report

Fontbakery version: 0.7.30

<details>
<summary><b>[6] Ballet[opsz].ttf</b></summary>
<details>
<summary>💔 <b>ERROR:</b> Check METADATA.pb font weights are correct.</summary>

* [com.google.fonts/check/metadata/os2_weightclass](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/os2_weightclass)
<pre>--- Rationale ---

Check METADATA.pb font weights are correct.

For static fonts, the metadata weight should be the same as the static font&#x27;s
OS/2 usWeightClass.

For variable fonts, the weight value should be 400 if the font&#x27;s wght axis
range includes 400, otherwise it should be the value closest to 400.


</pre>

* 💔 **ERROR** Failed with NameError: name 'f' is not defined

</details>
<details>
<summary>⚠ <b>WARN:</b> Description strings in the name table must not exceed 200 characters.</summary>

* [com.google.fonts/check/name/description_max_length](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/description_max_length)
<pre>--- Rationale ---

An old FontLab version had a bug which caused it to store copyright notices in
nameID 10 entries.

In order to detect those and distinguish them from actual legitimate usage of
this name table entry, we expect that such strings do not exceed a reasonable
length of 200 chars.

Longer strings are likely instances of the FontLab bug.


</pre>

* ⚠ **WARN** A few name table entries with ID=10 (NameID.DESCRIPTION) are longer than 200 characters. Please check whether those entries are copyright notices mistakenly stored in the description string entries by a bug in an old FontLab version. If that's the case, then such copyright notices must be removed from these entries. [code: too-long]

</details>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Fontfamily is listed on Google Fonts API?</summary>

* [com.google.fonts/check/metadata/listed_on_gfonts](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/listed_on_gfonts)

* ⚠ **WARN** Family not found via Google Fonts API. [code: not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Are there caret positions declared for every ligature?</summary>

* [com.google.fonts/check/ligature_carets](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/ligature_carets)
<pre>--- Rationale ---

All ligatures in a font must have corresponding caret (text cursor) positions
defined in the GDEF table, otherwhise, users may experience issues with caret
rendering.

If using GlyphsApp, ligature carets can be set directly on canvas by accessing
the `Glyph -&gt; Set Anchors` menu option or by pressing the `Cmd+U` keyboard
shortcut.


</pre>

* ⚠ **WARN** This font lacks caret position values for ligature glyphs on its GDEF table. [code: lacks-caret-pos]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
<pre>--- Rationale ---

Fonts with ligatures should have kerning on the corresponding non-ligated
sequences for text where ligatures aren&#x27;t used (eg
https://github.com/impallari/Raleway/issues/14).


</pre>

* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + i
	- i + f
	- f + l
	- l + f
	- i + l
	- b + b
	- d + d
	- g + g
	- l + l
	- p + p
	- r + r
	- t + t
	- z + z

   [code: lacks-kern-info]

</details>
<details>
<summary>⚠ <b>WARN:</b> The variable font 'opsz' (Optical Size) axis coordinate should be between 9 and 13 on the 'Regular' instance.</summary>

* [com.google.fonts/check/varfont/regular_opsz_coord](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/fvar.html#com.google.fonts/check/varfont/regular_opsz_coord)
<pre>--- Rationale ---

According to the Open-Type spec&#x27;s registered design-variation tag &#x27;opsz&#x27;
available at
https://docs.microsoft.com/en-gb/typography/opentype/spec/dvaraxistag_opsz

If a variable font has a &#x27;opsz&#x27; (Optical Size) axis, then the coordinate of its
&#x27;Regular&#x27; instance is recommended to be a value in the range 9 to 13.


</pre>

* ⚠ **WARN** The "opsz" (Optical Size) coordinate on the "Regular" instance is recommended to be a value in the range 9 to 13. Got 16.0 instead. [code: out-of-range]

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 1 | 0 | 5 | 38 | 8 | 129 | 0 |
| 1% | 0% | 3% | 21% | 4% | 71% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **INFO**
* **PASS**
* **DEBUG**


## Diff images: [report-pr.zip](http://35.238.63.0/media/2a478e9e-ec66-4f6a-b7df-f00c5f87254c/report-pr.zip)

**Comment by m4rc1e** [comment]
LGTM

File: ofl/ballet/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+date_added: "2020-09-24"
+fonts {
+  name: "Ballet"
+  style: "normal"
+  weight: 400
+  filename: "Ballet[opsz].ttf"
+  post_script_name: "Ballet-Regular"
+  full_name: "Ballet Regular"
+  copyright: "Copyright 2020 The Ballet Project Authors (https://github.com/Omnibus-Type/Ballet)"
+}
```
Comment (davelab6) [inline ofl/ballet/METADATA.pb:14]: This is missing the axes data

Comment (RosaWagner) [reply ofl/ballet/METADATA.pb:14]: there is no weight axis and the opsz is took of because not served by GF

Comment (davelab6) [reply ofl/ballet/METADATA.pb:14]: `opsz` is served by GF API (you can see some listed in fonts.google.com/variablefonts) but not surfaced for direct discovery in the main fonts.google.com UI.

Comment (RosaWagner) [reply ofl/ballet/METADATA.pb:14]: Ok cool, fixed here https://github.com/google/fonts/pull/2716

**Pull request merged**